src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -908,27 +913,41 @@ def setup(self, builder: Builder):
         self.clock = builder.time.clock()
         self.randomness = builder.randomness.get_stream(self.name)
 
+    def register_observations(self, builder: Builder) -> None:
+        # While this is a concatenating observation, it is not registered as such
+        # because we need to modify the per-time-step results_gatherer method
+        builder.results.register_unstratified_observation(
+            name=self.output_name,
+            pop_filter="",  # want to include untracked simulants
```

Comment:
I feel like this should be the default for this arg in vivarium.

Reply:
We need to have a larger discussion about changing fundamental defaults like this. I don't think any of us like the whole tracked thing anyway...